src/manifolds/Torus.jl

```diff
@@ -6,8 +6,8 @@ The n-dimensionsl torus is the $n$-dimensional product of the [´Circle`](@ref).
 The Circle is stored internally within `manifold`, such that all functions of
```

Comment:
See comments above on documenting the `PowerManifold` examples.

Reply:
I've added a simple example. Would like to also have something else here?

Reply:
Yes, looks nice, great.